proto/viam/app/v1/app.proto

```diff
@@ -60,6 +60,7 @@ message Robot {
   string name = 2 [(tagger.v1.tags) = "bson:\"name\" json:\"name\""];
   string location = 3 [(tagger.v1.tags) = "bson:\"location\" json:\"location\""];
   google.protobuf.Timestamp last_access = 4 [(tagger.v1.tags) = "bson:\"last_access\" json:\"last_access\""];
+  google.protobuf.Timestamp created_on = 5 [(tagger.v1.tags) = "bson:\"created_on\" json:\"created_on\""];
```

Comment:
We shouldn't be adding this when the field name is the same. Of course it follows the above precedence but we should remove the json tag here. Its automatically added by the proto-gen-go.

- Same goes for all other others.

More background in: https://docs.google.com/document/d/1UYNnrx8-
15G5KjuxsO215lUKMxna6FEyecEsh55q9sA/edit